src/components/load-more-stories-base.js

```diff
@@ -74,7 +74,14 @@ export class LoadMoreStoriesBase extends React.Component {
 export class LoadMoreCollectionStories extends React.Component {
   loadMoreStories(pageNumber) {
     const stories = get(this.props, ['data', 'stories'], []);
-    return getRequest(`/api/v1/collections/${this.props.collectionSlug}`, Object.assign({}, this.props.params, {
+    const collectionType = get(this.props, ["collectionType"],null);
+    const collectionId = get(this.props, ["collectionId"],null);
+    
+    const url = (collectionType && collectionId) ? 
+                `/api/v1/${collectionType}/${collectionId}/collection`: 
```

Comment:
Its not `collectionId`, its `authorId`. I guess passing authorId and checking that, instead of passing `collectionType ` should suffice.